core/src/sigverify_stage.rs

```diff
@@ -199,13 +228,22 @@ impl SigVerifyStage {
             timing::timestamp(),
             num_packets,
         );
+
+        let mut dedup_time = Measure::start("sigverify_dedup_time");
+        let dedup_fail = dedup_packets(bloom, &mut batches) as usize;
+        dedup_time.stop();
+
         let mut discard_time = Measure::start("sigverify_discard_time");
         if num_packets > MAX_SIGVERIFY_BATCH {
-            Self::discard_excess_packets(&mut batches, MAX_SIGVERIFY_BATCH);
-        }
+            Self::discard_excess_packets(&mut batches, MAX_SIGVERIFY_BATCH)
+        };
+        let excess_fail = (num_packets.saturating_sub(count_packets_in_batches(&batches)))
```

Comment:
@sakridge is this what you mean?

Reply:
can be `num_packets - dedup_fail - MAX_SIGVERIFY_BATCH`, no?